modules/serverpod_auth/serverpod_auth_server/lib/src/business/user_images.dart

```diff
@@ -64,14 +64,16 @@ class UserImages {
   static Future<bool> _setUserImage(
       Session session, int userId, ByteData imageData) async {
     // Find the latest version of the user image if any.
-    var oldImageRef = await session.db.findSingleRow<UserImage>(
-      where: UserImage.t.userId.equals(userId),
+    var oldImageRefs = await UserImage.db.find(
```

Comment:
Shouldn't this be a `findRow` call?

That way you don't have to handle a list bellow and the old logic stays the same.